decidim-proposals/spec/controllers/decidim/admin/proposal_notes_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Proposals
+    module Admin
+      describe ProposalNotesController, type: :controller do
```

Comment:
I'd rather have feature specs than controller specs.

Reply:
then, in this case is necessary the controller spec?

Reply:
I'm afraid no, it wouldn't be needed.